x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/workplace_search_gate.tsx

```diff
@@ -0,0 +1,751 @@
+            values={{
+              blogUrl: (
+                <EuiLink
+                  data-test-subj="workplaceSearch-gateForm-blog-link"
+                  href="#"
+                  target="_blank"
+                  data-telemetry-id="workplaceSearch-gateForm-blog-viewLink"
+                >
+                  {i18n.translate('xpack.enterpriseSearch.workplaceSearch.gateForm.viewBlog', {
+                    defaultMessage: 'blog',
+                  })}
+                </EuiLink>
+              ),
+            }}
+          />
+        ),
+        pageTitle: i18n.translate('xpack.enterpriseSearch.workplaceSearch.gateForm.title', {
+          defaultMessage: 'Before you begin...',
+        }),
+      }}
+      pageViewTelemetry="Workplace Search Gate form"
+      isLoading={isLoading}
+    >
+      <EuiPanel hasShadow={false}>
+        <EuiForm component="form" fullWidth>
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiFormLabel>
+                {i18n.translate('xpack.enterpriseSearch.workplaceSearch.gateForm.features.Label', {
+                  defaultMessage: 'What Workplace Search feature are you looking to use?',
+                })}
+              </EuiFormLabel>
+            </EuiFlexItem>
+            <EuiFlexGroup justifyContent="flexEnd">
+              <EuiFlexItem grow={false}>
+                <EuiText color="danger" size="xs">
+                  <p>
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.workplaceSearch.gateForm.features.required',
+                      {
+                        defaultMessage: 'Required',
+                      }
+                    )}
+                  </p>
+                </EuiText>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexGroup>
+          <EuiSpacer size="xs" />
+          <EuiSuperSelect
+            options={options}
+            valueOfSelected={feature}
+            placeholder={i18n.translate(
+              'xpack.enterpriseSearch.workplaceSearch.gateForm.features.selectOption',
+              {
+                defaultMessage: 'Select an option',
+              }
+            )}
+            onChange={(value) => setFeature(value)}
+            itemLayoutAlign="top"
+            hasDividers
+            fullWidth
+          />
+
+          {feature && <EducationPanel featureContent={feature} />}
+          <EuiSpacer />
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiFormLabel>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.workplaceSearch.gateForm.additionalFeedback.Label',
+                  {
+                    defaultMessage: 'Would you like to share any additional feedback?',
+                  }
+                )}
+              </EuiFormLabel>
+            </EuiFlexItem>
+            <EuiFlexGroup justifyContent="flexEnd">
```

Comment:
cc: @julianrosado Would it be okay to`Required` and `Optional` label?

Reply:
@saarikabhasi @sphilipse We need to label which fields are required at the very least so folks know what they can skip. Can we drop the optional tags but keep required? Because not tags is _madness_.

Reply:
Works for me!

Reply:
See above!